src/payment/strategies/barclaycard/barclaycard.ts

```diff
@@ -0,0 +1,13 @@
+export interface BrowserInfoRequest {
+    device_os: DeviceOS;
```

Comment:
I think the intention of the SDK is to keep all the fields using `camelCase`, however they should be converted to `snake_case`  in https://github.com/bigcommerce/bigpay-client-js/blob/7f163b5aa6c2ec4b18a0b12890cdde3a569c2118/src/payment/offsite-payment-mappers/payload-mapper.js#L67